### PR TITLE
fix(FEC-13939): Add Plugins Availability detection & fix style

### PR DESCRIPTION
**Pull request by JonathanTGold**

### CODE REVIEW

File: src/components/audio-player-view/audio-player-view.tsx [hunk 1]
```diff
@@ -242,7 +244,7 @@ const AudioPlayerView = Event.withEventManager(
               {hasError ? <ErrorSlate /> : _renderView()}
               {showPluginsMenuOverlay && (
                 <PluginsMenuOverlay
-                  poster={player.sources.poster}
+                  plugins={availablePlugins}
```
Comment (SivanA-Kaltura) [inline src/components/audio-player-view/audio-player-view.tsx:247]: what about poster ?

File: src/audio-player.tsx [hunk 2]
```diff
@@ -27,6 +32,12 @@ class AudioPlayer extends BasePlugin {
     this.prepareUI();
   }
 
+  loadMedia(): void {
+    Object.keys(this.player.plugins).forEach((plgName: string) => {
```
Comment (SivanA-Kaltura) [inline src/audio-player.tsx:36]: if loadMedia is called again the same plugins will be added twice

File: src/components/audio-player-view/audio-player-view.scss [hunk 3]
```diff
@@ -68,11 +68,9 @@ $ma-player-border-radius: var(--ma-player-border-radius);
...
       height: 100%;
       width: 100%;
-      filter: blur(48px) brightness(70%);
-      -webkit-filter: blur(48px) brightness(70%);
-      background-position: center;
-      background-repeat: no-repeat;
-      background-size: cover;
+      filter: blur(24px);
+      opacity: 0.5;
+      background: lightgray 50% / cover no-repeat;
```
Comment (SivanA-Kaltura) [inline src/components/audio-player-view/audio-player-view.scss:73]: why remove the brightness ?

**Review by SivanA-Kaltura — CHANGES_REQUESTED** [review]

File: src/components/audio-player-view/audio-player-view.tsx (hunk 1 above)
Comment (JonathanTGold) [reply src/components/audio-player-view/audio-player-view.tsx:247]: is is not nedded

File: src/components/audio-player-view/audio-player-view.scss (hunk 3 above)
Comment (JonathanTGold) [reply src/components/audio-player-view/audio-player-view.scss:73]: the style was not aligned with the requirements on figme, i took the style as it is from the figma on dev mode

File: src/components/audio-player-view/audio-player-view.tsx (hunk 1 above)
Comment (SivanA-Kaltura) [reply src/components/audio-player-view/audio-player-view.tsx:247]: why ?

File: src/audio-player.tsx (hunk 2 above)
Comment (JonathanTGold) [reply src/audio-player.tsx:36]: good point fixed (reset on reset())

File: src/components/audio-player-view/audio-player-view.tsx (hunk 1 above)
Comment (JonathanTGold) [reply src/components/audio-player-view/audio-player-view.tsx:247]: i don’t understand your questions , it is not needed in the component, please be more specific

File: src/components/audio-player-view/audio-player-view.scss (hunk 3 above)
Comment (JonathanTGold) [reply src/components/audio-player-view/audio-player-view.scss:73]: not got you, that’s exactly what I’m saying that it was not according to the spec and that’s why i fixed it to be according to the spec (and after talking with dana)

Comment (SivanA-Kaltura) [reply src/components/audio-player-view/audio-player-view.scss:73]: ok

**Review by SivanA-Kaltura — DISMISSED** [review]

File: package.json [hunk 4]
```diff
@@ -72,9 +72,7 @@
     "cypress": "^13.7.3"
   },
   "peerDependencies": {
-    "@playkit-js/kaltura-player-js": "3.17.15-canary.0-b412bfc",
-    "@playkit-js/playkit-js": "canary",
-    "@playkit-js/playkit-js-ui": "canary",
+    "@playkit-js/kaltura-player-js": "3.17.16-canary.0-db12b53",
```
[the remark this answers is not in the document]
Comment (JonathanTGold) [reply package.json:75]: Nothing, it is configuration since it is included in Kaltura-player-js, “peer dependency”

**Pull request merged**